src/interfaces/simulation.ts

```diff
@@ -275,7 +304,12 @@ export class SimulationInterface<T extends ChainId> extends ServiceInterface<T>
       value
     );
 
-    const [decimals, pricePerShare] = await Promise.all([vaultContract.decimals(), vaultContract.pricePerShare()]);
+    const [decimals, pricePerShare] = await Promise.all([
+      vaultContract.decimals(),
+      vaultContract.pricePerShare()
+    ]).catch(() => {
+      throw new EthersError("zap in");
```

Comment:
Is this a web3 error instead of a zap error?

Reply:
have specified the errors more